.circleci/config.yml

```diff
@@ -21,20 +21,32 @@ commands:
     parameters:
       method:
         type: string
-        default: "bootstrap"
+        default: "install"
     steps:
-      - restore_cache:
-          keys:
-            - v1-dependencies-{{ checksum "package.json" }}
-            # fallback to using the latest cache if no exact match is found
-            - v1-dependencies-
+      # - restore_cache:
```

Comment:
Are you saving these commented bits or can they be deleted?

Reply:
I plan on putting them back. Unfortunately the default cache behavior that circleci provides in their examples is far too simple for our needs so I had to remove it for now.